app-frontend/src/app/components/scenes/sceneDetailModal/sceneDetailModal.module.js

```diff
@@ -153,6 +174,26 @@ class SceneDetailModalController {
             this.newFilterFields[field] = this.getMaxBound(field);
         }
     }
+
+    selectDatasource(item) {
+        this.selectedDatasource = item;
+    }
+
+    saveDatasourceEdit() {
+        this.scene = Object.assign(this.scene, {
+            datasource: this.selectedDatasource.id,
+            'modifiedAt': this.moment().toISOString(),
```

Comment:
These fields (`modifiedAt` and `modifiedBy`) are automatically updated in the API, so you probably don't need to do this

Reply:
Yeah, I believe we ignore changes to these fields on the backend to prevent someone from inadvertently messing with these fields